### PR TITLE
Feature/total cloud cover

### PR DESCRIPTION
**Pull request by louisPoulain**
This PR aims at introducing post-processing of cloud cover. It notably includes:
- new NWP variables for the networks' inputs
- new "observations" (satellite data) as target

### CODE REVIEW

File: mlpp_features/sat.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+_LOGGER = logging.getLogger(__name__)
+
+# Set global options
+xr.set_options(keep_attrs=True)
+
+
+@out_format()
+def cloud_area_fraction(
```
Comment (dnerini) [inline mlpp_features/sat.py:17]: ask spc if we keep this name or prefer to name it `total_cloud_cover`

File: mlpp_features/sat.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    data: Dict[str, xr.Dataset], stations, reftimes, leadtimes, **kwargs
+) -> xr.DataArray:
+    """
+    Ensemble of total cloud cover in %
```
Comment (dnerini) [inline mlpp_features/sat.py:21]: it's a fraction between 0 and 1, check if this needs to be converted to % [0,100]

File: mlpp_features/sat.py
Comment (dnerini) [inline mlpp_features/sat.py:1]: shouldn't we rather add this to the `obs.py`?

File: mlpp_features/sat.py (hunk 1 above)
Comment (louisPoulain) [reply mlpp_features/sat.py:17]: Use standard CF: `cloud_area_fraction`

File: mlpp_features/sat.py (hunk 2 above)
Comment (louisPoulain) [reply mlpp_features/sat.py:21]: In spc's output files: in % but with a scale factore of 0.01, so we can keep it as a fraction between and 0 and 1

**Review by dnerini — APPROVED** [review]

**Pull request merged**